### ISSUE TITLE
View: Open Next Recently Used Editor in Group no longer works

### ISSUE THREAD

**Issue by Randalphwa** [microsoft/vscode#90753]
Issue Type: <b>Bug</b>

This stopped working in the latest update (1.42.1), and does not work from a key assignment or the command list.

Repo steps:
1. Open two files.
2. Manualy click on one and then the other files.
3. Press CTRL+SHIFT+P (or whatever key brings up Command list).
4: Enter View: Open Recently Used Editor in Group

Result: Currently selected editor remains selected.
Expected: The other editor should have gained focus.

VS Code version: Code 1.42.1 (c47d83b293181d9be64f27ff093689e8e7aed054, 2020-02-11T14:45:59.656Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz (8 x 4008)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.85GB (18.98GB free)|
|Process Argv|-r .|
|Screen Reader|no|
|VM|67%|
</details><details><summary>Extensions (18)</summary>

Extension|Author (truncated)|Version
---|---|---
numbered-bookmarks|ale|7.0.0
project-manager|ale|10.10.0
spellright|ban|3.0.50
markdown-preview-github-styles|bie|0.1.6
bracket-pair-colorizer-2|Coe|0.0.29
macros|ctf|0.0.4
gitlens|eam|10.2.1
seito-openfile|Fr4|1.8.5
ninja-syntax|mel|1.0.0
language-gettext|mro|0.1.1
cpptools|ms-|0.26.3
vs-keybindings|ms-|0.2.0
command-variable|rio|0.9.0
action-buttons|seu|1.1.5
cmake|twx|0.0.17
vscode-wakatime|Wak|2.3.0
clang-format|xav|1.9.0
markdown-all-in-one|yzh|2.7.0

</details>

**Comment by AndrewJRichardson** [comment]
Having the same issue.

Looks like the command may have been renamed to `Quick Open Previous Recently Used Editor In Group` there are now: 

`Open Previous Recently Used Editor In Group` (this one instantly jumps to the last used file before the current one)

and

`Quick Open Previous Recently Used Editor In Group` (shows the drop down where you can pick a file)

As far as I can tell `Quick Open Previous Recently Used Editor In Group ` has the same functionality as `Open Next Recently Used Editor In Group`. Though I might be wrong.

`Open Next Recently Used Editor In Group` is still a valid command but appears to do nothing now. Nothing in patch notes about this command as far as I can see.

**Comment by andreiashu** [comment]
experiencing the same issue.

For now I just assigned `Quick Open Previous Recently Used Editor In Group` to my usual shortcut and things are back to normal (thanks @AndrewJRichardson )

**Comment by bpasero** [comment]
Please read https://code.visualstudio.com/updates/v1_42#_navigate-most-recently-used-editors for understanding the changes. If there is still an issue, please speak up.

**Comment by Randalphwa** [comment]
I have switched the key assignment and verified that the new command appears to work identically to the previous command.

Closing since it's not technically a bug, but rather requires the user to change a key binding. I might argue that a change requiring a user to change their key binding should be displayed more prominently in the update notes, but that's a general comment, not a specific issue.
